extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -98,6 +98,10 @@ protected function requestParsoid( $method, $title, $params ) {
 			}
 		} elseif ( $status->isGood() ) {
 			$this->dieUsage( $req->getContent(), 'parsoidserver-http-' . $req->getStatus() );
```

Comment:
AFAIR `dieUsage` throws an exception. Please double check that the `WikiaLogger` code is actually executed here.